src/user/redux/modules/change_password.js

```diff
@@ -1,12 +1,20 @@
-import { FETCHED_MEMBER } from '../../../shared/redux/modules/member.js'
+import { createAction } from 'redux-actions'
```

Comment:
Will get on with this now. I was wondering though, what benefits does flow provide? I don't think I have experienced them...